tsconfig.json

```
@@ -13,7 +13,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "incremental": true
+    "incremental": true,
+    "experimentalDecorators": true
```

```suggestion
    "incremental": true
```

Is there a reason for this change? I can build the website fine without this compiler option.